routes/web.php

```diff
@@ -1,12 +1,17 @@
 <?php
 
 use App\Http\Controllers\PublicController;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Redirect;
 use Illuminate\Support\Facades\Route;
+use Illuminate\Http\Request;
 
 Route::get('/', function () {
     return view('welcome');
 });
 
 
 Route::get('index',[PublicController::class,'index'])->name('index');
-Route::get('about',[PublicController::class,'about'])->name('about');
\ No newline at end of file
+Route::get('about',[PublicController::class,'about'])->name('about');
+Route::get('404',[PublicController::class,'four0four'])->name('Four0four');
```

Comment:
no need for such a route as laravel by default load a view from `errors/404.blade.php` that what you just provide